assets/css/base/gutenberg-blocks.scss

```diff
@@ -680,3 +680,11 @@
 		}
 	}
 }
+
+// Top rated products
+.wc-block-grid {
+	img {
+		display: block;
```

Comment:
I think we can remove this rule, at least it seems to be always overridden by `.editor-styles-wrapper img` for me.
```suggestion
```